app/javascript/onboarding/components/SlideContent.jsx

```diff
@@ -12,17 +17,17 @@ const SlideContent = ({ imageSource, imageAlt, content, style = { textAlign: 'ce
     {content}
     <p>
       <strong>
-        <em>Let's get started...</em>
+        <em>Let`&apos;`s get started...</em>
```

Comment:
We don't need the backticks here, if we leave them in, it shows up like this: 
![image](https://user-images.githubusercontent.com/11466782/66509268-4b412580-ea98-11e9-983b-bea48beebcf2.png)